weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/ManageColumnsButton.tsx

```diff
@@ -0,0 +1,192 @@
+          variant="ghost"
+          icon="column"
+          tooltip="Manage columns"
+          onClick={onClick}
+        />
+      </span>
+      <Popover
+        id={id}
+        open={open}
+        anchorEl={anchorEl}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'center',
+        }}
+        transformOrigin={{
+          vertical: 'top',
+          horizontal: 'center',
+        }}
+        slotProps={{
+          paper: {
+            sx: {
+              overflow: 'visible',
+            },
+          },
+        }}
+        onClose={() => setAnchorEl(null)}
+        TransitionComponent={DraggableGrow}>
+        <Tailwind>
+          <div className="min-w-[300px] p-12">
+            <div className="handle flex items-center pb-8">
+              <div className="flex-auto font-semibold">Manage columns</div>
+              <div>
+                <Button
+                  size="small"
+                  variant="ghost"
+                  icon="close"
+                  tooltip="Close column management"
+                  onClick={e => {
+                    e.stopPropagation();
+                    setAnchorEl(null);
+                  }}
+                />
+              </div>
+            </div>
+            <div className="mb-8">
+              <TextField
+                placeholder="Filter columns"
+                autoFocus
+                value={search}
+                onChange={setSearch}
+              />
+            </div>
+            <div className="max-h-[300px] overflow-auto">
+              {columnInfo.cols.map((col: GridColDef<TraceCallSchema>) => {
+                const value = col.field;
+                const idSwitch = `toggle-vis_${value}`;
+                const checked = columnVisibilityModel[col.field] ?? true;
+                const label = col.headerName ?? value;
+                const disabled = !(col.hideable ?? true);
+                if (
+                  search &&
+                  !label.toLowerCase().includes(search.toLowerCase())
+                ) {
+                  return null;
+                }
+                return (
+                  <div key={value}>
+                    <div
+                      className={classNames(
+                        'flex items-center py-2',
+                        disabled ? 'opacity-40' : ''
+                      )}>
+                      <Switch.Root
+                        id={idSwitch}
+                        size="small"
+                        checked={checked}
+                        onCheckedChange={isOn => {
+                          toggleColumnShow(col.field);
+                        }}
+                        disabled={disabled}>
+                        <Switch.Thumb size="small" checked={checked} />
+                      </Switch.Root>
+                      <label
+                        htmlFor={idSwitch}
+                        className={classNames(
+                          'ml-4',
```

Comment:
Mega nit: feels the tiniest bit tight. not sure if everyone agrees. 

<img width="401" alt="Screenshot 2024-07-09 at 9 14 27 PM" src="https://github.com/wandb/weave/assets/19414170/0b748973-f9a0-4014-bfc3-f1f461f29576">

Reply:
compared to the before image it does now appear like there is less of a gap: 
![image](https://github.com/wandb/weave/assets/19414170/62f316d5-d79d-460c-bc80-6cad62d8e1ac)

Reply:
I'll bump it to `ml-6`, we can increase it more if it still feels tight.